src/VirtoCommerce.Platform.Web/wwwroot/js/app/assets/blades/asset-upload.js

```diff
@@ -64,15 +64,12 @@ angular.module('platformWebApp')
                     bladeNavigationService.setError(item._file.name + ' failed: ' + (response.message ? response.message : status), blade);
                 };
 
-                uploader.onCompleteAll = function () {
-                    refreshParentBlade();
-                    blade.uploadCompleted = true;
-                };
-
                 uploader.onSuccessItem = function (fileItem, images) {
+                    refreshParentBlade();
                     if (blade.onUploadComplete) {
                         blade.onUploadComplete(images);
                     }
+                    blade.uploadCompleted = true;
```

Comment:
why this change was needed? Comment in code, pls

Reply:
This is a bug. As I saw onCompleteAll always called. Therefore no chance to see an error. The user always sees the message about successful upload.